site/content/en/docs/user-manual/command-reference/transform.md

```diff
@@ -189,36 +250,404 @@ There are five methods of sampling (the `-m/--method` option):
+unlikely to be split exactly in the specified ratio.
+This split tries to split dataset images as close as possible to the specified
+ratio, keeping the initial class distribution.
+
+- reidentification splits
+In this task, the test set should consist of images of unseen people or
+objects during the training phase.
+This function splits a dataset in the following way:
+1. Splits the dataset into `train + val` and `test` sets
+  based on person or object ID.
+2. Splits `test` set into `test-gallery` and `test-query` sets
+  in class-wise manner.
+3. Splits the `train + val` set into `train` and `val` sets
+  in the same way.
+The final subsets would be `train`, `val`, `test-gallery` and `test-query`.
+
+Notes:
+- Each image is expected to have only one `Annotation`. Unlabeled or
+  multi-labeled images will be split into subsets randomly.
+- If Labels also have attributes, also splits by attribute values.
+- If there is not enough images in some class or attributes group,
+  the split ratio can't be guaranteed.
+
+In reidentification task,
+- Object ID can be described by Label, or by attribute (`--attr` parameter)
+- The splits of the test set are controlled by `--query` parameter
+  Gallery ratio would be `1.0 - query`.
+
+Usage:
+```bash
+split [-h] [-t {classification,detection,segmentation,reid}]
+  [-s SPLITS] [--query QUERY] [--attr ATTR_FOR_ID] [--seed SEED]
+```
+
+Optional arguments:
+- `-h`, `--help` (flag) - Show this help message and exit
+- `-t`, `--task` (one of: `classification`, `detection`, `segmentation`,
+  `reid`) - Dataset task (default: `classification`)
+- `-s`, `--subset` (str; repeatable) - Subsets in the form: '<subset>:<ratio>'
+  (default: {`train`: 0.5, `val`: 0.2, `test`: 0.3})
+- `--query` (float) - Query ratio in the test set (default: 0.5)
+- `--attr` (str) - Attribute name representing the ID (default: use label)
+- `--seed`(int) - Random seed
+
+Example:
+```
+datum transform -t split -- -t classification \
+  --subset train:.5 --subset val:.2 --subset test:.3
+
+datum transform -t split -- -t detection \
+  --subset train:.5 --subset val:.2 --subset test:.3
+
+datum transform -t split -- -t segmentation \
+  --subset train:.5 --subset val:.2 --subset test:.3
+
+datum transform -t split -- -t reid \
+  --subset train:.5 --subset val:.2 --subset test:.3 --query .5
+```
+
+Example: use `person_id` attribute for splitting
+```bash
+datum transform -t split -- -t detection --attr person_id
+```
+
+##### `map_subsets` <a id="map_subsets-transform"></a>
+
+Renames subsets in the dataset.
+
+Usage:
+```bash
+map_subsets [-h] [-s MAPPING]
+```
+
+Optional arguments:
+- `-h`, `--help` (flag) - Show this help message and exit
+- `-s`, `--subset` (str; repeatable) - Subset mapping of the form: `src:dst`
+
+##### `remap_labels` <a id="remap_labels-transform"></a>
+
+Changes labels in the dataset.
+
+A label can be:
+- renamed (and joined with existing) -
+  when specified `--label <old_name>:<new_name>`
```

Comment:
```suggestion
  when `--label <old_name>:<new_name>` is specified
```

Reply:
Please update the other list items correspondingly.

Reply:
Done